lib/modules/manager/argocd/extract.spec.ts

```diff
@@ -74,6 +79,32 @@ describe('modules/manager/argocd/extract', () => {
             datasource: 'docker',
             depName: 'somecontainer.registry.io:443/some/image3',
           },
+          {
+            currentValue: '1.0.0',
+            datasource: 'docker',
+            depName: 'somecontainer.registry.io:443/some/image3',
+          },
+          {
+            currentValue: 'v1.2.0',
+            datasource: 'git-tags',
+            depName: 'https://git.example.com/foo/bar.git',
+          },
+          {
+            currentValue: '1.0.0',
+            datasource: 'docker',
+            depName: 'somecontainer.registry.io:443/some/image3',
+          },
+          {
+            currentValue: 'v1.2.0',
+            datasource: 'git-tags',
+            depName: 'https://git.example.com/foo/bar.git',
+          },
+          {
+            currentValue: '0.0.2',
+            datasource: 'helm',
+            depName: 'traefik',
+            registryUrls: ['gs://helm-charts-internal'],
```

Comment:
shouldn't `gs://` be messaged to `https://`?

Reply:
The current behaviour on `main` is that only `oci://` is replaced.

```typescript
// /lib/modules/manager/argocd/extract.ts#L77
let registryURL = source.repoURL.replace('oci://', '');
```